dotCMS/src/main/java/com/dotmarketing/common/reindex/IndexResourceHelper.java

```diff
@@ -0,0 +1,97 @@
+package com.dotmarketing.common.reindex;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.elasticsearch.cluster.health.ClusterIndexHealth;
+import com.dotcms.content.elasticsearch.business.ContentletIndexAPI;
+import com.dotcms.content.elasticsearch.business.ESIndexAPI;
+import com.dotcms.content.elasticsearch.business.IndexStats;
+import com.dotmarketing.business.APILocator;
+import com.google.common.collect.ImmutableList;
+import io.vavr.control.Try;
+
+
+public class IndexResourceHelper {
+
+    private static class IndexResourceHelperHolder{
+        private static IndexResourceHelper helper = new IndexResourceHelper();
+    }
+    ContentletIndexAPI idxApi = APILocator.getContentletIndexAPI();
+    ESIndexAPI esapi = APILocator.getESIndexAPI();
+    
+    private IndexResourceHelper() {
+        
+
+    }
+
+    
+    static IndexResourceHelper getInstance() {
+        return IndexResourceHelperHolder.helper;
+    }
+    
+
+
+    public List<Map<String,Object>> indexStatsList()  {
+
+
+        Map<String,ClusterIndexHealth> clusterHealth = esapi.getClusterHealth();
+        List<String> openIndicies=idxApi.listDotCMSIndices();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'openIndicies' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)